T'au Empire.cat

```diff
@@ -1598,7 +1598,11 @@
       <infoLinks>
         <infoLink id="a1ab-acf8-166c-61e2" name="New InfoLink" hidden="false" targetId="9a3a-31af-3eb4-fc3d" type="rule"/>
         <infoLink id="db8d-fb86-ed8d-5d09" name="Drone Support" hidden="false" targetId="9b19-f096-5c9d-696d" type="rule"/>
-        <infoLink id="87e1-0f1b-a43a-2840" name="Vanguard" hidden="false" targetId="0090-7776-c531-10b8" type="profile"/>
+        <infoLink id="87e1-0f1b-a43a-2840" name="Vanguard" hidden="false" targetId="0090-7776-c531-10b8" type="profile">
+          <modifiers>
+            <modifier type="set" field="21befb24-fc85-4f52-a745-64b2e48f8228" value="At the start of the first battle round but before the first turn begins, you can move this unit and any accompanying Drones up to 7&quot;. They cannot end this move within 9&quot; of any enemy models. If both players have units that are able to move at the start of the first battle round but before the first turn begins, the player who is taking the first turn moves their units first."/>
```

Comment:
Second sentence: `They`, should be `It`

Reply:
In book: "they". Codex Tau, pg.107. But will change if you want :)

Reply:
Oh. Well, I thought the OP from here: https://github.com/BSData/wh40k/issues/5022 checked xD Sorry!